lib/ActivityStreams.js

```diff
@@ -75,6 +80,27 @@ ActivityStreams.prototype = {
   },
 
   /**
+   * Every 6 hours pro-actively prime our emebedly cache with a normal content page load
+   */
+  _primeCache() {
+    setTimeout(function() {
+      PlacesProvider.links.getTopFrecentSites.then(links => {
+        PreviewProvider.preview.primeCache(links);
+      });
+      PlacesProvider.links.getRecentBookmarks.then(links => {
+        PreviewProvider.preview.primeCache(links);
+      });
+      PlacesProvider.links.getRecentLinks.then(links => {
+        PreviewProvider.preview.primeCache(links);
+      });
+      PlacesProvider.links.getFrecentLinks.then(links => {
+        PreviewProvider.preview.primeCache(links);
+      });
+      this._primeCache();
+    }.bind(this), PRIME_CACHE_TIMEOUT);
```

Comment:
Not sure if we could switch to an arrow function here to avoid needing to use `.bind(this)`. :shrug: